### DISCUSSION TITLE
JK_PB2A16S20P: RJ45 Ground wire is not enough explained :(

### DISCUSSION THREAD

**Discussion by ldiegos** [txubelaxu/esphome-jk-bms#18] [Show and tell]
Hi Txube,

After a few hours trying to identify my problems, untill I realize that the RJ45 cable needs to connect also a GND wire... is not explained enough, is not present in the pictures. Hope you can update the pictures in order to show that the third wire should be connected.

**Comment by Sleeper85** [comment]
[I made more precise schematic for my project.](https://github.com/Sleeper85/esphome-jk-bms-can/blob/development/documents/README/BMS_JK-PB_RS485_solution.md)

**Comment by txubelaxu** [comment]
I have just updated the schematics: https://github.com/txubelaxu/esphome-jk-bms/blob/main/components/jk_rs485_bms/README.md

I think it is clearer now.
